packages/ui/components/button/Button.tsx

```diff
@@ -210,7 +210,7 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
             <StartIcon
               className={classNames(
                 variant === "icon" && "h-4 w-4",
-                variant === "button" && "h-4 w-4 stroke-[1.5px] ltr:mr-2 rtl:ml-2"
+                variant === "button" && "h-4 w-4 stroke-[1.5px]"
```

Comment:
Breaks icon buttons in our app - Please check storybook and [design system](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=13743%3A97381&t=6OjJIg9y55PcPjn2-0) when making changes to core components